core/server/data/migrations/versions/4.33/02-add-default-free-product.js

```diff
@@ -0,0 +1,37 @@
+const {createTransactionalMigration} = require('../../utils');
+const ObjectID = require('bson-objectid');
+const {slugify} = require('@tryghost/string');
+const logging = require('@tryghost/logging');
+
+module.exports = createTransactionalMigration(
+    async function up(knex) {
+        const [result] = await knex
+            .count('id', {as: 'total'})
+            .from('products')
+            .where('type', 'free');
+
+        if (result.total !== 0) {
+            logging.warn(`Not adding default free tier, a free tier already exists`);
+            return;
+        }
+
+        const name = 'Free';
+
+        logging.info(`Adding tier "${name}"`);
+        await knex('products')
+            .insert({
+                id: ObjectID().toHexString(),
+                name: name,
+                description: 'Free Preview',
+                type: 'free',
+                slug: slugify(name),
+                created_at: knex.raw(`CURRENT_TIMESTAMP`)
```

Comment:
Can't remember if we need a `created_by` too - do we?

Reply:
We don't use `created_by` anymore in the newer schemas, and `products` table doesn't has any such column so we are good to skip it.